### PR TITLE
Ok call

### PR DESCRIPTION
**Pull request by ntoll**
This PR (stacked upon #25) adds a single OK function:

* To be used to confirm success of a command.
* Displays optional message.
* Green signifies success.

Looks like this in action:

ok

### CODE REVIEW

File: tests/test_generator.py
Comment (fpliger) [inline tests/test_generator.py:23]: Not sure I get this assert...

**Review by fpliger — APPROVED** [review]
Overall LGTM. Could use some more tests but can come in a follow up PR. Only comment I had was about an assert but PR looks good! TY!

File: src/pyscript/cli.py
Comment (mattkram) [inline src/pyscript/cli.py:20]: I think this would print `"OK. None"` if no `msg` was provided. We should probably change the default to:
```
def ok(msg: str = ""):
```

and then to be pedantic about trailing whitespace:
```
console.print(f"OK. {msg}".rstrip(), style="green")
```

File: tests/test_generator.py
Comment (mattkram) [reply tests/test_generator.py:23]: This code is actually from #25 . I have a proposed refactor I've pushed to remove mocks and test explicitly.

**Comment by mattkram** [comment]
Changed the base branch. It will revert to `main` once #25 is merged.

**Comment by fpliger** [comment]
Awesome. Thank you @mattkram ! I'll let @ntoll merge this and #25 tomorrow.

**Comment by ntoll** [comment]
Ugh git. All fixed up. Will merge when things turn green.

**Pull request merged**
